resources/scss/findologic.scss

```diff
@@ -0,0 +1,97 @@
+.findologic-container {
```

Comment:
Delete this SCSS file and keep the CSS version of it. There is a separate story for converting everything.